services/token_create.proto

```diff
@@ -191,4 +191,9 @@ message TokenCreateTransactionBody {
      * The custom fees to be assessed during a CryptoTransfer that transfers units of this token
      */
     repeated CustomFee custom_fees = 21;
+
+    /**
+     * The Key which can pause and unpause the Token. If Empty the token is unPaused by default.
```

Comment:
```suggestion
     * The Key which can pause and unpause the Token. If Empty the token is unpaused by default.
```

Reply:
Would there ever be a reason to have a token default to paused by default, similar to how there is a `freezeDefault`?

Reply:
Had a discussion in today's scrum. Decided not necessary in the current scope.